config/chef/data_bags/users/deploy.json.example

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "deploy",
+  "comment": "Deploy user",
+  "shell": "/bin/bash",
+
+  // TODO: Include instructions on encrypting a password
+  "password": "REPLACE ME",
+
+  "ssh_keys": [
+    "REPLACE ME"
+  ]
+}
```

Comment:
If this is using the `opengov-users` cookbook, I think `deploy` is gonna get stepped on by the `deploy` user that that sets up.  I'd recommend using a different default user name here?